.travis.yml

```diff
@@ -15,9 +15,9 @@ before_install:
 
 install:
   - docker run -v /usr/local/bin:/target jpetazzo/nsenter
-  - go install -v $(go list ./... | grep -v /vendor/)
   - hack/install-protoc.sh
   - hack/install-hypercontainer.sh
+  - go install -v $(go list ./... | grep -v /vendor/)
```

Comment:
why this change?

Reply:
@feiskyer I've introduce libvirt-go dependency in this patch, when go install `libvirt-go`, it will try to build it, and it need libvirt-dev lib to do that. We installed this lib in `hack/install-hypercontainer.sh`, so I changed this order to install libvirt-dev first. 

BTW, new patch is on the way, thanks for reviewing.

Reply:
OK, that makes sense. thanks